src/policy/fees.cpp

```diff
@@ -9,11 +9,17 @@
 #include <clientversion.h>
 #include <primitives/transaction.h>
 #include <streams.h>
-#include <txmempool.h>
 #include <util.h>
 
 static constexpr double INF_FEERATE = 1e99;
 
+struct CBlockPolicyEstimator::Block {
+    Block(int height) : height(height) {}
```

Comment:
```
2018-09-20 04:48:56 clang-tidy(pr=10443): src/policy/fees.cpp:17:5: warning: single-argument constructors must be marked explicit to avoid unintentional implicit conversions [google-explicit-constructor]
```